src/adapter/src/coord.rs

```diff
@@ -1737,6 +1737,15 @@ impl Coordinator {
         let write_frontier = self.least_valid_write(&id_bundle);
         // Things go wrong if we try to create a dataflow with `as_of = []`, so avoid that.
         if write_frontier.is_empty() {
+            tracing::info!(
+                export_ids = dataflow.format_export_ids(),
+                %cluster_id,
+                min_as_of = ?min_as_of.elements(),
+                write_frontier = ?write_frontier.elements(),
```

Comment:
@teskje why are we logging `write_frontier` here if we are in an `if` block says that `write_frontier.is_empty()`?